gradle.properties

```diff
@@ -6,7 +6,7 @@
 # http://www.gradle.org/docs/current/userguide/build_environment.html
 # Specifies the JVM arguments used for the daemon process.
 # The setting is particularly useful for tweaking memory settings.
-org.gradle.jvmargs=-Xmx2048m
+org.gradle.jvmargs=-Xmx2048m -Dfile.encoding=UTF-8
```

Comment:
@ChristianoBolla-Appwise What is this?

Reply:
Hmm... it's something created by Android Studio itself... 🤔 
I'll delete it for now!! 😄